admin/admin-service/src/main/java/io/mosip/admin/constant/AdminErrorCode.java

```diff
@@ -0,0 +1,24 @@
+package io.mosip.admin.constant;
+
+public enum AdminErrorCode {
+
+    UNABLE_TO_RETRIEVE_RID("ADM-AVD-001", "A technical error occurred while retrieving the data, please try again after some time."),
```

Comment:
Rename this FROM UNABLE_TO_RETRIEVE_RID to UNABLE_TO_RETRIEVE_RID_DETAILS

Reply:
taken care

Reply:
Resolved